### PR TITLE
Add code style check in travis CI.

### PR DESCRIPTION
**Pull request by tz70s**
Simply add to existing job, due to it's not time consuming in current codebase.

## Purpose

Style checker in Travis CI.

## References

References #610 

## Changes

`scalafmtCheckAll` for checking all sources and `scalafmtSbtCheck` for checking sbt build sources.

## Background Context

https://scalameta.org/scalafmt/docs/installation.html#task-keys

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ install: jabba install "$TRAVIS_JDK" && jabba use "$_" && java -Xmx32m -version
 
 jobs:
   include:
-    - script: sbt test headerCheck test:headerCheck grpcVersionSyncCheck
+    - script: sbt test headerCheck test:headerCheck grpcVersionSyncCheck scalafmtCheckAll scalafmtSbtCheck
```
Comment (tz70s) [inline .travis.yml]: Question:

I guess the `test` is not necessary here due to we already have 2x2 matrix for tests.
WDYT?

Comment (raboof) [reply .travis.yml]: I think the other jobs run integration tests, and this is the only job that runs the unit tests - unless I'm misunderstanding you?

**Comment by raboof** [comment]
OK TO TEST

File: .travis.yml (hunk 1 above)
Comment (tz70s) [reply .travis.yml]: The default Test stage has 2x2 matrix for running unit tests.
I think the `jobs include` add additional jobs to the default Test stage.
(https://docs.travis-ci.com/user/build-stages/#build-stages-and-build-matrix-expansion).

e.g. All 6.1 - 6.4 jobs run unit tests, and the 6.1 & 6.5 run the same test for same Scala version and same env (JDK version).
screenshot

**Pull request merged**